conda_self/cli/main_reset.py

```diff
@@ -1,24 +1,60 @@
 from __future__ import annotations
 
+from textwrap import dedent
 from typing import TYPE_CHECKING
 
 if TYPE_CHECKING:
     import argparse
 
 HELP = "Reset 'base' environment to essential packages only."
 
+WHAT_TO_EXPECT = dedent("""
+This will:
+1.Reset your `base` to ONLY contain `conda` (and its dependencies), `conda` plugins
+and `conda-self`.
+2.Protect the `base` from further modifications
```

Comment:
Does `conda self reset` protect the base environment? I thought it only removes non-essential packages.